.github/workflows/ci-test-playwright.yml

```
@@ -6,37 +6,38 @@ on:
     inputs:
       project:
         description: "Playwright project (smoke, sanity, regression, chromium). Empty = all."
-        required: false
+        required: 
```

actionlint (1.7.12)

[error] expected bool value but found scalar node with "!!null" tag

(syntax-check)